src/components/header.js

```diff
@@ -26,8 +26,8 @@ const Header = ({ color }) => {
     <React.Fragment>
 
       <div className="announcement-banner bg-[#302871] py-3">
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/argocon/">
-          ArgoCon 23 EU is a wrap. View recordings and presentations here.
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/argocon/">
+          ArgoCon NA 2023 is now a co-located event at KubeCon/CloudNativeCon 2023 Chicago. CFP is now open!
```

Comment:
Do we mean CFP is open for ArgoCon or CFP in general?  Looking at the KubeCon CFP pages, the button states `CFP CLOSED` [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/cfp/).

Reply:
Nevermind.  https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/argocon/ argo con has its own page (helps if I look a the link above. 🤦